### PR TITLE
Add example notebook

### PR DESCRIPTION
**Pull request by pnkraemer**
There is a very plain jupyter notebook with a tutorial now. Please dont be too critical of the content, it is just to get the ball rolling :) 

This PR also includes 
* notebook-to-md syncing via jupytext (has to be set up manually, see jax' contribution guide; let's write down somewhere what how why etc.)
* notebook linting via nbqa (included in tox)

### CODE REVIEW

**Review by nathanaelbosch — CHANGES_REQUESTED** [review]

File: docs/tutorial_notebooks/getting_started.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+# Create an ODE problem
+f, tspan, u0 = ivp_examples.vanderpol(stiffness_constant=1)
+
+print(f, tspan, u0)
+```
+
+Next, we choose a solver.
+There are a couple of suggestions. For example, let us solve the ODE for the terminal value and use the Kronecker EK0 (the most efficient solver we have).
+
+Solvers are tuples of an `init_fn` and a `perform_step_fn`, similar to how `optax` handles optimisers, and `blackjax` handles samplers. The `perform_step_fn` has error estimation and calibration baked into the implementation.
```
[the remark this answers is not in the document]
Comment (pnkraemer) [reply docs/tutorial_notebooks/getting_started.md:49]: Good idea! (I committed this locally, bc. of the syncing with markdown files.)

File: docs/tutorial_notebooks/getting_started.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+```python
+ek0_solver = ek0.ek0_kronecker_terminal_value(num_derivatives=4)
+
+print(ek0)
```
[the remark this answers is not in the document]
Comment (pnkraemer) [reply docs/tutorial_notebooks/getting_started.md]: removed.

**Comment by pnkraemer** [comment]
I included a notebook-execution check into tox and fixed the current issues with the notebook.

**Comment by pnkraemer** [comment]
Btw: we could even think about versioning _only_ the `*.md` files (which means git-ignoring the notebooks). See [here](https://jupytext.readthedocs.io/en/latest/examples.html#collaborating-on-jupyter-notebooks). The pro would be that there is only a single version to be tracked (and this whole jupytext-syncing-business is more self-explanatory). But this might remove some outputs, so for now I don't think it is really necessary.

**Review by nathanaelbosch — APPROVED** [review]
Looks good!

Regarding tracking only the .md's: It seems reasonable to me to only track the md files and not the notebooks. But I'm not familiar with the tooling, so I'm fine with whatever you think is sensible to do :)

**Pull request merged**